README.md

```diff
@@ -1,2 +1,38 @@
 # Cobalt-Cowboy
-An ESP32-C3 based radio running OpenRTX Firmware
+
+A RISC-V-based radio running OpenRTX Firmware
+
+The goal of this project is to make available a hackable radio development platform built using open-source hardware and firmware, reducing the copyright and patent overhead required to build radio equipment and technology.
+
+The intended users of this project include licenced Amateur Radio operators and students.
+
+This repo includes the PCB and Schematic and associated [KiCAD](https://www.kicad.org/) project files. In essence, a ESP32C3+CC1200 evaluation board built for use with the [M17 Project](https://m17project.org/) and capabilities of the [OpenRTX firmware](https://openrtx.org).
```

Comment:
I'd say something more like "This repo includes the PCB, schematic, and associated KiCAD project files *as well as a parts list and any information needed to build your own Cobalt Cowboy*..."